### ISSUE TITLE
Mobile Menu Icon Fails WCAG 2.0 Level AA Accessibility Scans

### ISSUE THREAD

**Issue by mcrowell** [elementor/elementor#13859]
- [x ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
The default mobile menu icon doesn't pass WCAG 2.0 Level AA accessibility scans.

**Describe the solution you'd like**
Give us the option to insert our own SVG icons or use the default i class icon.

**Additional context**
If you run a scan on https://achecker.ca/ you will see that the default mobile menu icon is causing a Known Problem on WCAG 2.0 Level AA scans. Below is an example of the problem:

![Elementor-Mobile-Menu-WCAG2-AA](https://user-images.githubusercontent.com/7768009/107375716-0af01e00-6ab7-11eb-9b73-9e0c76bc24ed.jpg)

**Comment by AleksBaikalski** [comment]
+1
icon fonts - it is the past, long gone. @elementor team, please make icon sets bundled to svg sprites and use them with `use` tag and `role="presentation"`

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.3.0

Feel free to update

Thanks!
